packages/xrpl/src/sugar/getOrderbook.ts

```diff
@@ -101,7 +82,30 @@ async function getOrderbook(
   if (options.taker !== undefined && typeof options.taker !== 'string') {
     throw new ValidationError('taker must be a string', options.taker)
   }
+}
 
+/**
+ * Creates a request object for retrieving book offers.
+ *
+ * @param currency1 - The first currency in the pair.
+ * @param currency2 - The second currency in the pair.
+ * @param options - Additional options for the request.
+ * @param [options.limit] - The maximum number of offers to retrieve.
+ * @param [options.ledger_index] - The ledger index to use for retrieval.
+ * @param [options.ledger_hash] - The ledger hash to use for retrieval.
+ * @param [options.taker] - The taker address for retrieval.
```

Comment:
Does adding the `[...]` brackets impact the generated docs? (It's not a convention I've been following, but can start following if it's helpful)

Reply:
That's how you denote optional in jsdoc